run-tests.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+cd terminado
+python -m unittest discover $@
```

Comment:
Thank, let's do that. I think it should be as simple as removing this file and adding to the README "run tests with `nosetests`".